### PR TITLE
Project list updates

### PR DESCRIPTION
**Pull request by solth**

### CODE REVIEW

File: Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml [hunk 1]
```diff
@@ -63,8 +63,8 @@
                             <li class="nav-pic-text"><h:link value="#{msgs.prozesse}" outcome="processes"><i class="fa fa-hourglass-3 fa-lg"/></h:link></li>
                             <li class="nav-pic-text"><h:link value="#{msgs.projekte}" outcome="projects"><i class="fa fa-folder-open fa-lg"/></h:link></li>
                             <li class="nav-pic-text"><h:link value="#{msgs.benutzer}" outcome="users"><i class="fa fa-user fa-lg"/></h:link></li>
-                            <li class="nav-pic-text"><h:link value="#{msgs.taskmanager}" outcome="newTaskManager"><i class="fa fa-puzzle-piece fa-lg"/></h:link></li>
```
Comment (henning-gerhardt) [inline Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml]: Is removing of this two lines intended?

Comment (solth) [reply Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml]: I didn't intend to remove them, but just to change the message tag used in the value attribute.

Comment (henning-gerhardt) [reply Kitodo/src/main/webapp/WEB-INF/templates/includes/header.xhtml]: Ok.

**Comment by solth** [comment]
Forgot to add some necessary changes, sorry for the late commit.

**Pull request merged**